ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1130,6 +1133,26 @@ export const WorkspaceEdit = fp.flow(
       this.saveWorkspace();
     }
 
+    getAskAboutAIAN() {
+      const publicCDRVersionNumber: number = getCdrVersion(
+        this.state.workspace,
+        this.props.cdrVersionTiersResponse
+      ).publicReleaseNumber;
+
+      return publicCDRVersionNumber >= 8;
```

Comment:
Do we want 8 to be a configurable value?

Reply:
To me, a config value implies that this will vary between environments or change in the future.  I don't think that's what we have here.  I would like to see it as a const, maybe at the top of this file?  Tests and stubs could reference that too.